README

```diff
@@ -34,9 +34,14 @@ II. Misc
 
 The layer contains three recipes for OpenDDS: two versioned, one not.
 
-To select the versioned recipes for OpenDDS 3.22, 3.20 or 3.14, include
+To select the versioned recipes for OpenDDS 3.24, 3.22, 3.20 or 3.14, include
 the following in your local.conf 
 
+  PREFERRED_VERSION_opendds = "3.24" and optionally set ACE_TAO_OPTION to one of the following:
+  ACE_TAO_OPTION = ""             to build with OCI DOC Group ACE 6.5.19 / TAO 2.5.19
```

Comment:
```suggestion
  ACE_TAO_OPTION = ""             to build with the DOC Group ACE 6.5.19 / TAO 2.5.19
```

Reply:
Change according to peer review comments: https://github.com/OpenDDS/meta-opendds/pull/23/commits/45abe3c2612a7ed7d644b44b9c21cbda328dcf75